### DISCUSSION TITLE
UpDate()가 작동하지 않습니다 .

### DISCUSSION THREAD

**Discussion by jefflim69** [naverz/zepeto-studio-kor#1074] [5. Q&A]
Log 로 확인 해 봐도 UpDate 가 작동 하지 않네요 .
다른 처리 방식이 있나요 ?
![UpDate_Error](https://user-images.githubusercontent.com/12184282/181065638-f21ffb64-df91-40fc-84c8-36c47199f3a1.PNG)

**Comment by chlguswls32** [comment]
안녕하세요. jefflim69님

UpDate -> Update 로 수정해주시면 아마 실행될것같아요. 
*대소문자때문에 안되는것같습니다.

**Reply by jefflim69** [reply]
감사합니다. 작동 합니다 . 
별 실수를 다하네요 .
